-------------------------------------------------------------------------------
Test set: com.KlasemenLiga.KlasemenLiga.RepoTimApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 3.911 s <<< FAILURE! - in com.KlasemenLiga.KlasemenLiga.RepoTimApplicationTests
contextLoads  Time elapsed: 0.007 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on com.KlasemenLiga.KlasemenLiga.Entity.EntityStatistikData.awayTeam references an unknown entity: java.lang.String
Caused by: org.hibernate.AnnotationException: @OneToOne or @ManyToOne on com.KlasemenLiga.KlasemenLiga.Entity.EntityStatistikData.awayTeam references an unknown entity: java.lang.String

